tools/stress-cluster/chaos/examples/parallel-pod-example/templates/parallel-pod.yaml

```diff
@@ -0,0 +1,18 @@
+{{- include "stress-test-addons.parallel-deploy-job-template.from-pod" (list . "stress.parallel-pod-example" 3) -}}
```

Comment:
a comment here saying 3 means 3 pods in parallel might be helpful for users

Reply:
@ckairen fixed

Reply:
nvm, the doc mentioned it, but I think keeping the doc and example consistrnt might be good. so like 3 or 5 but @ both places

Reply:
I'll change both to three.